### PR TITLE
Add gaming cursor mode (pointer lock) and a small bugfix

### PR DESCRIPTION
**Pull request by CorbinWunderlich**
Pointer lock makes gaming over Xpra finally usable and actually works pretty well. When cursor lock is turned on, it automatically locks you to the focused window (which counts the desktop in desktop mode), and when you exit with with the `esc` key, it pauses, and waits for you to click on a new window to lock on. The paused mode (and cursor lock) is disabled by default and can be disabled through the float menu, with some colors to indicate.

The small bugfix is just using my new css classes to color the keyboard in as well, as the previous approach to coloring it wasn't working as jQuery cannot change css `:before` attributes.

### CODE REVIEW

File: html5/js/Window.js [hunk 1]
```diff
@@ -351,13 +351,13 @@ class XpraWindow {
...
     });
-    canvas.addEventListener("pointermove", (event_) => {
+    canvas.addEventListener("mousemove", (event_) => {
       this.debug("mouse", "pointermove:", event_);
       if (this.pointer_down == event_.pointerId) {
         const dx = event_.offsetX - this.pointer_last_x;
         const dy = event_.offsetY - this.pointer_last_y;
         this.pointer_last_x = event_.offsetX;
-        this.pointer_last_y = event_.offsetY;
+        this.pointer_last_y = event_.offsetX;
```
Comment (totaam) [inline html5/js/Window.js:360]: Is this right?

File: html5/js/Window.js [hunk 2]
```diff
@@ -351,13 +351,13 @@ class XpraWindow {
         this.pointer_last_y = event_.offsetY;
       }
     });
-    canvas.addEventListener("pointermove", (event_) => {
+    canvas.addEventListener("mousemove", (event_) => {
       this.debug("mouse", "pointermove:", event_);
```
Comment (totaam) [inline html5/js/Window.js:355]: Please update the debug logging to match the code. 
BTW, why is this change needed here?

File: html5/js/Client.js [hunk 3]
```diff
@@ -326,7 +326,7 @@ class XpraClient {
     const screen_element = jQuery("#screen");
     screen_element.mousedown((e) => this.on_mousedown(e));
     screen_element.mouseup((e) => this.on_mouseup(e));
-    screen_element.mousemove((e) => this.on_mousemove(e));
+    document.getElementById("screen").addEventListener("mousemove", (e) => this.on_mousemove(e));
```
Comment (totaam) [inline html5/js/Client.js:329]: Why change this one and not the ones above it? 
What's the difference between them?

Comment (CorbinWunderlich) [reply html5/js/Client.js:329]: jQuery is too old, and does not support the movementX and movementY event variables that are needed for cursor lock

File: html5/js/Window.js (hunk 1 above)
Comment (CorbinWunderlich) [reply html5/js/Window.js:360]: No, that was my mistake. I'll fix this

File: html5/js/Window.js (hunk 2 above)
Comment (CorbinWunderlich) [reply html5/js/Window.js:355]: I am pretty sure this is necessary for cursor lock, as pointermove changes when the pointer is moved (does not happen when locked), and mousemove changes when the mouse is moved (correct). I will update the logging. Sorry.
